src/org/opendatakit/briefcase/util/AggregateUtils.java

```diff
@@ -264,26 +243,21 @@ private static final DocumentFetchResult httpRetrieveXmlDocument(HttpUriRequest
           WebUtils.resetHttpContext();
           ex = new XmlDocumentFetchException("Authentication failure");
         } else if (statusCode == 400) {
-          ex = new XmlDocumentFetchException(description.getFetchDocFailed() + webError + " while accessing: "
-              + uri.toString() + "\nPlease verify that the " + description.getDocumentDescriptionType()
-              + " that is being uploaded is well-formed.");
+          ex = new XmlDocumentFetchException("" +
```

Comment:
I don’t see the value of the `"" +`, here and below.

Reply:
I thought you would like it! It lets next lines to be correctly indented and aligned.

Reply:
You are the human. You can align the lines below however you like. :-)

Reply:
But the IDE does that! Not the human! :D